### ISSUE TITLE
Bus master interface with PLIN-USB

### ISSUE THREAD

**Issue by Chowdarypvr** [rbei-etas/busmaster#1179]
Does anyone tried using Bus master for LIN control using PLIN-USB hardware from Peak?
Tool says "Listed Hardware interface is failed" so wondering if the drivers of this device are being integrated into this tool or not? Does it even work with any of the Peak LIN hardwares?

**Comment by geier99** [comment]
Yes I tried it.  I use the PCAN-USB-Pro hardware and it's working well.

**Comment by janw-cz** [comment]
See https://github.com/rbei-etas/busmaster/issues/1188 (with patch file).
